conjur/controller/resource_controller.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+"""
+ResourceController module
+
+This module is the controller that facilitates all resource actions
+required to successfully execute the RESOURCE command
+"""
+
+import json
+import sys
+from conjur.logic.resource_logic import ResourceLogic
+from conjur.resource import Resource
+
+# pylint: disable=too-few-public-methods
+class ResourceController:
+    """
+    ResourceController
+
+    This class represents the Presentation Layer for the RESOURCE command
+    """
+
+    def __init__(self, resource_logic: ResourceLogic):
+        self.resource_logic = resource_logic
+
+    def exists(self, resource_id: str, json_response: str = False):
+        """
+        Method that facilitates the exists command
+        """
+        resource = Resource.from_full_id(resource_id)
+        result = self.resource_logic.exists(resource.kind, resource.identifier)
+
+        if json_response:
+            sys.stdout.write(f"{json.dumps({'exists' : result}, indent=4)}\n")
```

Comment:
I saw your recent PR added a util_functions.print_json_result. If we want to be consistent and use it for all json outputs in controller methods, I think you could incorporate it here like so: 
```suggestion            
           print_json_result({'exists' : result})
```

Reply:
Thanks for reminding me! I forgot the code I wrote two days ago :facepalm: